docs/systems/important/communication-system.md

```
@@ -0,0 +1,381 @@
+| Field | Purpose |
+|-------|---------|
+| `hide` | Hides social from room observers when target present |
+| `minPos` | Minimum position required for target |
+| `char_no_arg` | Message to actor when no target |
+| `others_no_arg` | Message to room when no target |
+| `char_found` | Message to actor when target found |
+| `others_found` | Message to room when target found |
+| `vict_found` | Message to target |
+| `not_found` | Message when target not found |
+| `char_auto` | Message to actor for self-targeting |
+| `others_auto` | Message to room for self-targeting |
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)